docs/DeveloperGuide.md

```diff
@@ -213,8 +305,8 @@ The following sequence diagram shows how a Find operation goes through the `Logi
 **Aspect: Tag Parsing in FindCommandParser:** <br>
 
 * **Alternative 1: (Current Choice):** Parsing tags separately from other search criteria.
-  * Pros: Allows users to specify additional search criteria based on tags.
-  * Cons: Requires additional parsing logic and potentially increases complexity in handling multiple search parameters.
+  * Pros: Allows users to specify Deleteitional search criteria based on tags.
+  * Cons: Requires Deleteitional parsing logic and potentially increases complexity in handling multiple search parameters.
```

Comment:
This part may have been accidentally overwritten when replacing add with delete.

Reply:
Right, edited!